tinygrad/runtime/ops_metal.py

```diff
@@ -1,12 +1,13 @@
 # pip3 install pyobjc-framework-Metal pyobjc-framework-Cocoa pyobjc-framework-libdispatch
 import os, subprocess, pathlib, functools, ctypes
 import Metal, Cocoa, libdispatch # type: ignore
-from typing import List, Any
+from typing import List, Any, Tuple
 from tinygrad.codegen.kernel import LinearizerOptions
 from tinygrad.renderer.cstyle import uops_to_cstyle, CStyleLanguage
-from tinygrad.helpers import prod, getenv, DEBUG, DType, dtypes
-from tinygrad.ops import Compiled
+from tinygrad.helpers import prod, getenv, DEBUG, DType, dtypes, GlobalCounters
+from tinygrad.ops import Compiled, ASTRunner, BasicBatchExecutor
 from tinygrad.runtime.lib import RawBufferMapped, LRUAllocator
+from tinygrad.shape.symbolic import sym_infer
```

Comment:
The backend should not be importing this

Reply:
removed